ledger/ledger-on-sql/src/main/scala/com/daml/ledger/on/sql/queries/TimedQueries.scala

```diff
@@ -35,6 +35,10 @@ final class TimedQueries(delegate: Queries, metricRegistry: MetricRegistry) exte
   override def updateState(stateUpdates: Seq[(Key, Value)]): Try[Unit] =
     Timed.value(Metrics.updateState, delegate.updateState(stateUpdates))
 
+  override def truncate(): Try[Unit] = {
+    Timed.value(Metrics.truncate, delegate.truncate())
+  }
```

Comment:
I assume this is now fast enough that we can re-enable the tests?

Reply:
Re-enabled in b4d5a7f.

Reply:
@SamirTalwar-DA: I guess we can now also re-enable `ResetServiceInMemoryIT` for sandbox?

Reply:
Already done in #5778.